### ISSUE TITLE
[1.19.2] Crashes With Create 0.5.1b

### ISSUE THREAD

**Issue by MarioAndWeegee3** [Kneelawk/WiredRedstone#37]
```
[22:33:20] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'wiredredstone'!
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$2(EntrypointUtils.java:66) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:34) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:64) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invokeContainer(EntrypointUtils.java:49) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:65) ~[quilt-loader-0.19.0.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) [transformed-mod-minecraft-v1.19.2.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:524) [quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:82) [quilt-loader-0.19.0.jar:?]
	at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:31) [quilt-loader-0.19.0.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
	at org.quiltmc.loader.impl.launch.server.QuiltServerLauncher.main(QuiltServerLauncher.java:66) [quilt-loader-0.19.0.jar:?]
	Suppressed: java.lang.NoClassDefFoundError: com/simibubi/create/content/contraptions/wrench/IWrenchable
		at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
		at java.lang.ClassLoader.defineClass(ClassLoader.java:1013) ~[?:?]
		at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:310) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:299) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClassOnly(KnotClassDelegate.java:164) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:150) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:228) ~[quilt-loader-0.19.0.jar:?]
		at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
		at com.github.talrey.createdeco.registry.MetalDecoBuilders.buildCatwalk(MetalDecoBuilders.java:274) ~[transformed-mod-createdeco.i0:0/:?]
		at com.github.talrey.createdeco.Registration.lambda$registerBlocks$51(Registration.java:503) ~[transformed-mod-createdeco.i0:0/:?]
		at java.util.HashMap.forEach(HashMap.java:1429) ~[?:?]
		at com.github.talrey.createdeco.Registration.registerBlocks(Registration.java:454) ~[transformed-mod-createdeco.i0:0/:?]
		at com.github.talrey.createdeco.CreateDecoMod.onInitialize(CreateDecoMod.java:43) ~[transformed-mod-createdeco.i0:0/:?]
		at org.quiltmc.loader.impl.game.minecraft.Hooks.lambda$startServer$2(Hooks.java:65) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke$0(EntrypointUtils.java:36) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:62) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invokeContainer(EntrypointUtils.java:49) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:65) ~[quilt-loader-0.19.0.jar:?]
		at net.minecraft.server.Main.main(Main.java:101) [transformed-mod-minecraft-v1.19.2.jar:?]
		at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
		at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
		at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:524) [quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:82) [quilt-loader-0.19.0.jar:?]
		at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:31) [quilt-loader-0.19.0.jar:?]
		at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104) ~[?:?]
		at java.lang.reflect.Method.invoke(Method.java:578) ~[?:?]
		at org.quiltmc.loader.impl.launch.server.QuiltServerLauncher.main(QuiltServerLauncher.java:66) [quilt-loader-0.19.0.jar:?]
	Caused by: java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.wrench.IWrenchable
		at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
		at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
		at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClassOnly(KnotClassDelegate.java:178) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:150) ~[quilt-loader-0.19.0.jar:?]
		at org.quiltmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:228) ~[quilt-loader-0.19.0.jar:?]
		at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
		... 29 more
Caused by: java.lang.NoClassDefFoundError: com/simibubi/create/content/contraptions/components/structureMovement/BlockMovementChecks$MovementNecessaryCheck
	at com.kneelawk.wiredredstone.compat.create.CreateCompatImpl.init(CreateCompatImpl.kt:11) ~[transformed-mod-wiredredstone.i0:0/:?]
	at com.kneelawk.wiredredstone.compat.create.CreateCompatHandler.init(CreateCompatHandler.kt:12) ~[transformed-mod-wiredredstone.i0:0/:?]
	at com.kneelawk.wiredredstone.WiredRedstoneModKt.init(WiredRedstoneMod.kt:36) ~[transformed-mod-wiredredstone.i0:0/:?]
	at java.lang.invoke.MethodHandleProxies$1.invoke(MethodHandleProxies.java:199) ~[?:?]
	at com.sun.proxy.jdk.proxy3.$Proxy29.onInitialize(Unknown Source) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.Hooks.lambda$startServer$2(Hooks.java:65) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke$0(EntrypointUtils.java:36) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:62) ~[quilt-loader-0.19.0.jar:?]
	... 12 more
Caused by: java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.structureMovement.BlockMovementChecks$MovementNecessaryCheck
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClassOnly(KnotClassDelegate.java:178) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:150) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:228) ~[quilt-loader-0.19.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	at com.kneelawk.wiredredstone.compat.create.CreateCompatImpl.init(CreateCompatImpl.kt:11) ~[transformed-mod-wiredredstone.i0:0/:?]
	at com.kneelawk.wiredredstone.compat.create.CreateCompatHandler.init(CreateCompatHandler.kt:12) ~[transformed-mod-wiredredstone.i0:0/:?]
	at com.kneelawk.wiredredstone.WiredRedstoneModKt.init(WiredRedstoneMod.kt:36) ~[transformed-mod-wiredredstone.i0:0/:?]
	at java.lang.invoke.MethodHandleProxies$1.invoke(MethodHandleProxies.java:199) ~[?:?]
	at com.sun.proxy.jdk.proxy3.$Proxy29.onInitialize(Unknown Source) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.Hooks.lambda$startServer$2(Hooks.java:65) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke$0(EntrypointUtils.java:36) ~[quilt-loader-0.19.0.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:62) ~[quilt-loader-0.19.0.jar:?]
	... 12 more
```

**Comment by Kneelawk** [comment]
Oh boy, I'll see if I can get a fix for that out soon.

**Comment by Kneelawk** [comment]
Ok, this should be fixed in version `0.4.18` on [Modrinth](https://modrinth.com/mod/wiredredstone/version/0.4.18+1.19.2) and [Curseforge](https://legacy.curseforge.com/minecraft/mc-mods/wired-redstone/files/4599196).
